packages/tempo/public/esm.index.html

```
@@ -0,0 +1,18 @@
+<script type="importmap">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)